CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changes
 4.3.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Editing while reading.
```

Comment:
I don't find this very informative. Perhaps something more like "Correct typos and ReST formatting errors in documentation."